### PR TITLE
WX-854 Use supported Terra region in test

### PR DESCRIPTION
**Pull request by jgainerdewar**
Ticket: [<Link to Jira ticket>](https://broadworkbench.atlassian.net/browse/WX-854)

These tests started failing because when trying to run the workflow, we couldn't write to the bucket. This changes the region to one supported by Terra.

---

**PR checklist**

- [x] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [x] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

### CODE REVIEW

**Comment by jgainerdewar** [comment]
jenkins retest

**Comment by jgainerdewar** [comment]
jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala [hunk 1]
```diff
@@ -247,7 +247,7 @@ class RawlsApiSpec
       }(owner.makeAuthToken(billingScopes))
     }
 
-    "should be able to create workspace and run sub-workflow tasks in non-US regions" taggedAs(MethodsTest) in {
+    "should be able to create workspace and run sub-workflow tasks in non-US regions" taggedAs(MethodsTest) ignore in {
```
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala]: change `ignore in` to just `ignore`, I think:
```suggestion
    "should be able to create workspace and run sub-workflow tasks in non-US regions" taggedAs(MethodsTest) ignore {
```

**Review by davidangb — APPROVED** [review]

**Review by aednichols — APPROVED** [review]
Looks OK to me on the basis that we do not publicly claim support for European regions [0] 

The purpose of this test is to verify that we launch compute in the same region as the workspace bucket, to avoid egress. Its purpose is not specifically to check whether compute works in Finland.

[0] https://support.terra.bio/hc/en-us/articles/360058964552-Customizing-where-your-data-are-stored-and-analyzed

**Pull request merged**